lib/lambda_ethereum_consensus/state_transition/epoch_processing.ex

```diff
@@ -0,0 +1,44 @@
+defmodule LambdaEthereumConsensus.StateTransition.EpochProcessing do
+  @moduledoc """
+  Update effective balances with hysteresis
+  """
+
+  alias ChainSpec
+  alias SszTypes.BeaconState
+  alias SszTypes.Validator
+
+  @spec process_effective_balance_updates(BeaconState.t()) :: :ok
```

Comment:
Thank you for your contribution @ayushm2003 ! So a couple of things. The signature of this function should return `{:ok, BeaconState.t()}  |  {:error, binary() }`. In Elixir, values are immutable, so you can't change the state, but you can create a new one and return it.

Reply:
ahh, I assumed it returned the last value in the function, so since `LambdaEthereumConsensus.Store.StateStore.store_state(new_state)` returns `:ok` I went with it. Will correct